templates/macros.xml.j2

```diff
@@ -2,38 +2,9 @@
 <!-- {{ ansible_managed }} -->
 <yandex>
     <macros>
-        <cluster>{{ clickhouse_display_name | default('clickhouse') }}</cluster>
-        {% if inventory_hostname in groups['clickhouse_shard01'] %}
-        <shard>01</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard02'] %}
-        <shard>02</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard03'] %}
-        <shard>03</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard04'] %}
-        <shard>04</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard05'] %}
-        <shard>05</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard06'] %}
-        <shard>06</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard07'] %}
-        <shard>07</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard08'] %}
-        <shard>08</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard09'] %}
-        <shard>09</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard10'] %}
-        <shard>10</shard>
-        {% endif %}
+        <cluster>{{ cluster }}</cluster>
+        <shard>{{ shard }}</shard>
```

Comment:
should not we have for loop here?

Reply:
yes we could do something like
```
{% for k, v in macros.items() %}
    <{{ k }}>{{ v }}</{{ k }}>
{% endfor %}
```
Then in host_vars:

```
macros:
  key1: value1
  key2: value2
```
WDYT?